docs/install.md

```diff
@@ -0,0 +1,70 @@
+# Installing the Knative Pipelines component
+
+Before you can run Knative Pipelines, you must install the Knative Pipelines components
+in your Kubernetes cluster. Use this page to add the component to an existing 
+Kubernetes cluster.
+
+## Before you begin
+
+For complete installation instructions of Knative components, see [Installing Knative](https://github.com/knative/docs/blob/master/install/README.md).
+
+## Adding the Knative Pipelines component
+
+To add only the Knative Pipelines component to an existing installation:
+
+1. Run the
+   [`kubectl apply`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply)
+   command to install [Knative Pipelines](https://github.com/knative/build-pipeline) and its
+   dependencies:
+   ```bash
+   kubectl apply --filename https://storage.googleapis.com/knative-releases/build-pipeline/latest/release.yaml
+   ```
+1. Run the
+   [`kubectl get`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#get)
+   command to monitor the Knative Pipelines components until all of the components
+   show a `STATUS` of `Running`:
+
+   ```bash
+   kubectl get pods --namespace knative-build-pipeline
+   ```
+
+   Tip: Instead of running the `kubectl get` command multiple times, you can
+   append the `--watch` flag to view the component's status updates in real
+   time. Use CTRL + C to exit watch mode.
+
+You are now ready to create and run Knative Pipelines: 
+ - See [Knative Pipeline tutorial](./tutorial.md) to get started. 
+ - Look at [the examples](https://github.com/knative/build-pipeline/tree/master/examples)
+
+
+## Configuring Knative Pipelines
+
+### How are resources shared between tasks?
+
+Pipelines need a way to share resources between tasks. The alternatives are a [Persistent volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)
+or a (GCS storage bucket)[https://cloud.google.com/storage/]
+
+The PVC option does not require any configuration, but the GCS storage bucket can be 
+configured using a ConfigMap with the name `config-artifact-bucket` with the following 
+attributes:
+
+- location: the address of the bucket (for example gs://mybucket)
+- bucket.service.account.secret.name: the name of the secret that will contain
+  the credentials for the service account with access to the bucket
+- bucket.service.account.secret.key: the key in the secret with the required
+  service account json. 
+- The bucket is recommended to be configured with a
+  retention policy after which files will be deleted.
+
+Both options provide the same functionality to the pipeline. The choice is based
+on the infrastructure used, for example in some Kubernetes platforms, the
+creation of a persistent volume could be slower than uploading/downloading files
+to a bucket, or if the the cluster is running in multiple zones, the access to
+the persistent volume can fail.
```

Comment:
nice!! we should add something here about the entrypoint configuration too, which I think I removed by accident when i re-organized the docs :sweat_smile: https://github.com/knative/build-pipeline/blob/6269b7a216992cf51d370eebc2ca473233049b13/docs/using.md#configure-entrypoint-image

Reply:
will add it